results.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="3" errors="0" failures="0" skip="0"><testcase classname="integration.smoke.test_enable_role_based_users_in_projects.TestRoleBasedUsersInProjects" name="test_add_account_to_project_with_project_role" time="3.046"></testcase><testcase classname="integration.smoke.test_enable_role_based_users_in_projects.TestRoleBasedUsersInProjects" name="test_add_multiple_admins_in_project" time="6.491"><system-out><![CDATA[pid == 3a16742f-819b-4d44-89d9-1f4122602038
+2
+]]></system-out></testcase><testcase classname="integration.smoke.test_enable_role_based_users_in_projects.TestRoleBasedUsersInProjects" name="test_add_user_to_project_with_project_role" time="2.713"></testcase></testsuite>
```

Comment:
this file is missing a license. either add it to the ignore list or add a license.

Reply:
I see this results.xml is changed now but still without license. I think it makes sense to either remove it from git or add it to the exclusion list for the rat, @Pearl1594 .